src/runtime/nuxt2.plugin.ts

```diff
@@ -0,0 +1,36 @@
+import { defineNuxtPlugin, useRuntimeConfig } from '#app'
+import { useRouter } from "#imports";
+
+export default defineNuxtPlugin(async (_) => {
+	const { id, metrikaUrl, ...metrikaOptions } = useRuntimeConfig().yandexMetrika
+
+	let ready = false
+	const basePath = (useRouter().options.base || '/').replace(/\/$/, '')
```

Comment:
we can likely use runtime config to get this instead, here and in the nuxt3 version (perhaps we could unify them?):

```js
import { useRuntimeConfig } from '#imports'
// ...
const basePath = useRuntimeConfig().app.basePath
```